### ISSUE TITLE
3.7.7 and 3.7.8 Migration Issue

### ISSUE THREAD

**Issue by Ardy000** [alexal1/Insomniac#314]
```
[04/08 14:53:49] Creating tables in insomniac.db...
[04/08 14:53:49] [Migration] Migrating sessions to the insomniac.db...
[04/08 14:53:50] [Migration] Migrating interacted users to the insomniac.db...

Caught an exception:
log_like_action() missing 2 required positional arguments: 'task_id' and 'execution_id'
[04/08 14:53:51] Traceback (most recent call last):
  File "<string>", line 196, in run
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/insomniac/migration.py", line 180, in migrate_from_sql_to_peewee
    my_profile.log_like_action(session_id, interacted_user["username"], None, None)
TypeError: log_like_action() missing 2 required positional arguments: 'task_id' and 'execution_id'

```
Does anyone know what this means?

**Comment by VincentAdam** [comment]
I have exactly the same problem.

**Comment by VincentAdam** [comment]
I have renamed my profile db folder, and reexecute the script. It's seems to be fine for 10 minutes and crash again with a new error "Cannot find the List of followers..." there is the crash test :

[Crash-2021-04-13-12-05-38.zip](https://github.com/alexal1/Insomniac/files/6302929/Crash-2021-04-13-12-05-38.zip)

**Comment by alexal1** [comment]
Fixed in `v3.7.15`
